components/micro-gateway-core/src/main/ballerina/gateway/services/throttle_event_listener.bal

```diff
@@ -19,56 +19,82 @@ import ballerina/log;
 import ballerina/io;
 import ballerina/config;
 import ballerina/jms;
+import ballerina/http;
 
-string jmsConnectioninitialContextFactory = getConfigValue(THROTTLE_CONF_INSTANCE_ID,
-    JMS_CONNECTION_INITIAL_CONTEXT_FACTORY,
-    "bmbInitialContextFactory");
+string jmsConnectionInitialContextFactory = getConfigValue(THROTTLE_CONF_INSTANCE_ID,
+    JMS_CONNECTION_INITIAL_CONTEXT_FACTORY, "bmbInitialContextFactory");
 string jmsConnectionProviderUrl = getConfigValue(THROTTLE_CONF_INSTANCE_ID, JMS_CONNECTION_PROVIDER_URL,
     "amqp://admin:admin@carbon/carbon?brokerlist='tcp://localhost:5672'");
 string jmsConnectionPassword = getConfigValue(THROTTLE_CONF_INSTANCE_ID, JMS_CONNECTION_PASSWORD, "");
 string jmsConnectionUsername = getConfigValue(THROTTLE_CONF_INSTANCE_ID, JMS_CONNECTION_USERNAME, "");
 
-// Initialize a JMS connection with the provider.
-jms:Connection jmsConnection = new({
-        initialContextFactory: jmsConnectioninitialContextFactory,
-        providerUrl: jmsConnectionProviderUrl,
-        username: jmsConnectionUsername,
-        password: jmsConnectionPassword
-    });
-
-// Initialize a JMS session on top of the created connection.
-jms:Session jmsSession = new(jmsConnection, {
-        acknowledgementMode: "AUTO_ACKNOWLEDGE"
-    });
-
-// Initialize a Topic subscriber using the created session.
-listener jms:TopicSubscriber subscriberEndpoint = new(jmsSession, topicPattern = "throttleData");
-
-// Bind the created consumer(subscriber endpoint) to the listener service.
-service jmsListener on subscriberEndpoint {
+service jmsListener =
+service {
     resource function onMessage(jms:TopicSubscriberCaller consumer, jms:Message message) {
-        var m = message.getMapMessageContent();
-        if(m is map<any>) {
+        map<any>|error m = message.getMapMessageContent();
+        if (m is map<any>) {
             log:printDebug("ThrottleMessage Received");
             //Throttling decisions made by TM going to throttleDataMap
             if (m.hasKey(THROTTLE_KEY)) {
-                GlobalThrottleStreamDTO globalThrottleStreamDtoTM = {};
-                globalThrottleStreamDtoTM.throttleKey = <string>m[THROTTLE_KEY];
-                globalThrottleStreamDtoTM.isThrottled = <boolean>m[IS_THROTTLED];
-                globalThrottleStreamDtoTM.expiryTimeStamp = <int>m[EXPIRY_TIMESTAMP];
+                GlobalThrottleStreamDTO globalThrottleStreamDtoTM = {
+                    throttleKey: <string>m[THROTTLE_KEY],
+                    isThrottled: <boolean>m[IS_THROTTLED],
+                    expiryTimeStamp: <int>m[EXPIRY_TIMESTAMP] };
 
                 if (globalThrottleStreamDtoTM.isThrottled == true) {
                     putThrottleData(globalThrottleStreamDtoTM);
-                } else {
+                }
+                else {
```

Comment:
Is there any reason to break the else statement to two lines, I noticed this in several places.

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/332/commits/7dec453bb0eb0d526d5b95004e51f7b012efb678